src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -878,7 +878,7 @@ public List<SourceLocation> FindPossibleBreakpoints(SourceLocation start, Source
             if (start.Id != end.Id)
             {
                 logger.LogDebug($"FindPossibleBreakpoints: documents differ (start: {start.Id}) (end {end.Id}");
-                return null;
+                throw new ReturnAsErrorException($"FindPossibleBreakpoints: documents differ (start: {start.Id}) (end {end.Id}");
```

Comment:
Is this the same error js throws?

Reply:
Probably not. I just surfaced the errors that we already had. I can check what js returns for that, and do the same thing.